cypress/support/pages/backoffice/ssp-service-management/product-abstract-type/product-abstract-type-repository.ts

```diff
@@ -0,0 +1,17 @@
+import { autoWired } from '@utils';
+import { injectable } from 'inversify';
+
+@injectable()
+@autoWired
+export class ProductAbstractTypeRepository {
```

Comment:
We already have repository for Edit Abstract Product
https://github.com/spryker/cypress-tests/blob/master/cypress/support/pages/backoffice/product-management/edit/product-management-edit-repository.ts